### PR TITLE
[schema] replace duplicated struct w/ google struct (schema files only)

### PR DESCRIPTION
**Pull request by oded-soffrin**

### CODE REVIEW

**Comment by oded-soffrin** [comment]
First phase solution:
having the wix-schema using google struct to unblock platformized efforts (auto-velo)
converting to current solution on OS to keep same behavior as today (bundle size, converters, etc)

next steps:
use google struct in OS as well and add mutate function to fix existing externalData content to match new format

File: packages/ricos-schema/web/src/generateSchema.ts [hunk 1]
```diff
@@ -35,6 +35,11 @@ const run = async () => {
         .replace('import "wix/api/entity.proto";\n', '')
         .replace('import "wix/api/validations.proto";\n', '')
         .replace('import "wix/api/annotations.proto";\n', '')
+        .replace(
+          'import "google/protobuf/struct.proto"',
```
Comment (lxgreen) [inline packages/ricos-schema/web/src/generateSchema.ts:39]: does this replacement mean we use google struct on wix-ricos only?

Comment (lxgreen) [reply packages/ricos-schema/web/src/generateSchema.ts:39]: if yes, then how can we update the draft converters?

**Review by lxgreen — APPROVED** [review]
so, AFAIU the OS has no changes here?

**Pull request merged**